### PR TITLE
(Developer) Nav updates

### PR DESCRIPTION
**Pull request by hestonhoffman**
### What does this PR do? What is the motivation?

See [DOCS-7748](https://datadoghq.atlassian.net/browse/DOCS-7748)

### Merge instructions

- [x] Please merge after reviewing

### Additional notes

[DOCS-7748]: https://datadoghq.atlassian.net/browse/DOCS-7748?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/developers/service_checks/_index.md [hunk 1]
```diff
@@ -38,12 +38,10 @@ Service checks can be visualized and used in 3 Datadog sections:
 
 ### Check summary
 
-Click **Monitors** > **Check Summary** to see the [Check Summary][1] page.
+The [Check Summary][1] page lists of all checks reported across your infrastructure in the past day. Select a check to get insights on the status and tags associated with the check.
```
Comment (hestonhoffman) [inline content/en/developers/service_checks/_index.md:41]:
```suggestion
The [Check Summary][1] page lists all checks reported across your infrastructure in the past day. Select a check to get insights on the status and tags associated with the check.
```

File: content/en/developers/integrations/create-an-integration-recommended-monitor.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ To create a Datadog integration, see [Create a New Integration][3].
 ## Steps to create a recommended monitor
 ### Build a monitor JSON Schema
 
-1. Navigate to **[Monitors > New Monitor][4]** and create a new monitor.
+1. [Create a monitor][5].
```
Comment (urseberry) [inline content/en/developers/integrations/create-an-integration-recommended-monitor.md:21]:
```suggestion
1. [Create a monitor][4].
```
Gremlins maybe changed the link number?

File: content/en/developers/integrations/create-a-cloud-siem-detection-rule.md [hunk 3]
```diff
@@ -21,7 +21,9 @@ To create a Datadog integration, see [Create a New Integration][3].
 ## Create a detection rule
 ### Build a detection rule JSON schema
 
-1. Navigate to **[Security > Cloud SIEM > Detection Rules > New Rule][4]** and create a new rule. Note: Only threshold based rules are currently supported.
+1. From **Detection Rules**, [create a new rule][4].
+
+   **Note**: Only threshold based rules are supported.
```
Comment (eho1307) [inline content/en/developers/integrations/create-a-cloud-siem-detection-rule.md:26]:
```suggestion
```
This note can be removed now

File: content/en/developers/integrations/log_pipeline.md [hunk 4]
```diff
@@ -64,11 +64,10 @@ Logs sent to Datadog are processed in [log pipelines][13] to standardize them fo
...
 
-1. Navigate to [**Logs** > **Pipelines**][3].
-2. Click **+ New Pipeline**.
-3. In the **Filter** field, enter a unique `source` tag that defines the log source for the Technology Partner's logs. For example, `source:okta` for the Okta integration. **Note**: Make sure that logs sent through the integration are tagged with the correct source tags before they are sent to Datadog.
-4. Optionally, add tags and a description.
-5. Click **Create**.
+1. From the [**Pipelines**][3] page, click **+ New Pipeline**.
+1. In the **Filter** field, enter a unique `source` tag that defines the log source for the Technology Partner's logs. For example, `source:okta` for the Okta integration. **Note**: Make sure that logs sent through the integration are tagged with the correct source tags before they are sent to Datadog.
+1. Optionally, add tags and a description.
+1. Click **Create**.
```
Comment (eho1307) [inline content/en/developers/integrations/log_pipeline.md:70]:
```suggestion
1. From the [**Pipelines**][3] page, click **+ New Pipeline**.
2. In the **Filter** field, enter a unique `source` tag that defines the log source for the Technology Partner's logs. For example, `source:okta` for the Okta integration. **Note**: Make sure that logs sent through the integration are tagged with the correct source tags before they are sent to Datadog.
3. Optionally, add tags and a description.
4. Click **Create**.
```

**Review by eho1307 — CHANGES_REQUESTED** [review]

**Review by eho1307 — APPROVED** [review]

**Review by urseberry — APPROVED** [review]

**Pull request merged**